ros/processor/inventory_events_consumer.py

```diff
@@ -97,21 +107,25 @@ def host_delete_event(self, msg):
                 self.prefix
             )
             rows_deleted = db.session.query(System.id).filter(System.inventory_id == host_id).delete()
+            db.session.commit()
             if rows_deleted > 0:
                 processor_requests_success.labels(
-                    reporter=self.reporter, account_number=msg['host']['account']
+                    reporter=self.reporter, account_number=msg['account']
                 ).inc()
                 LOG.info(
                     'Deleted host from inventory with id: %s - %s',
                     host_id,
                     self.prefix
                 )
-            db.session.commit()
 
     def host_create_update_events(self, msg):
         """ Process created/updated message ( create system record, store new report )"""
         self.prefix = "PROCESSING Create/Update EVENT"
         if 'is_ros' in msg['platform_metadata']:
+            LOG.info(
+                'Process a msg for host(%s) belonging to account %s - %s',
+                msg['host']['id'], msg['host']['account'], self.prefix
+            )
```

Comment:
I'm not sure IIUC, but does this log msg want to convey something like this:
```suggestion
            LOG.info(
                'Processing inventory events message for host(%s) belonging to account %s - %s',
                msg['host']['id'], msg['host']['account'], self.prefix
            )
```

Reply:
As we are already adding a prefix into message, skipping `inventory events` substring. Except inventory, ROS is not listening to any other service for create/updated event. I hope this fine!